tasks/gen.go

```diff
@@ -105,12 +105,27 @@ func main() {
 		schema.SpawnStructField("WorkedBy", "String", true, false),
 		schema.SpawnStructField("Stage", "String", false, false),
 		schema.SpawnStructField("CurrentStageDetails", "StageDetails", true, false),
+		schema.SpawnStructField("PastStageDetails", "List_StageDetails", true, false),
```

Comment:
@gammazero can answer this better, but I believe we do have this data already. It's just separate entries in the database, since it's like a journal. We have a unique constraint on:

> task_status_ledger.uuid, task_status_ledger.status, task_status_ledger.stage

If we only stored the last state, this constraint wouldn't make sense - we would simply have unique tasks by UUID.